src/wx/xrc/MainMenu.xrc

```diff
@@ -508,13 +508,21 @@
           <label>_Debug print</label>
           <checkable>1</checkable>
         </object>
+        <object class="wxMenuItem" name="GBALcdFilter">
+          <label>_LCD Filter</label>
+          <checkable>1</checkable>
+        </object>
       </object>
       <object class="wxMenu">
         <label>_Game Boy</label>
         <object class="wxMenuItem" name="GameBoyConfigure">
           <label>Configure ...</label>
         </object>
         <object class="separator"/>
+        <object class="wxMenuItem" name="GBColorOption">
+          <label>_GB color option</label>
```

Comment:
You have the accelerator `_G` already being used above, choose a different letter.

Reply:
See this:

https://docs.wxwidgets.org/3.0/overview_xrcformat.html#overview_xrcformat_type_text

Reply:
So this is the reason of these `_` before some letters... I will check it out.